### DISCUSSION TITLE
Which languages are supported other than English?

### DISCUSSION THREAD

**Discussion by bipul-mohanto** [mermaid-js/mermaid#4149] [Q&A]
Hi!
I was trying to write something with language Bengali and Hindi. But unfortunately, the compilation failed. Is it possible `mermaid` support for such U+0981 languages?

**Comment by bipul-mohanto** [comment]
I found Chinese, Japanese, and German are being supported. Could it be possible for the developers to take the South Asian language into consideration?

**Reply by sidharthv96** [reply]
Hi @bipul-mohanto, 
Currently, class & flow diagram parsers seems to have partial unicode support.

I'll convert this as an issue that's welcoming PRs.
